### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by TylerAHolden**
Couldn't find the npm package very easily. Keywords might help others too

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,5 +129,15 @@
...
+  "keywords": [
+    "stream",
+    "get",
+    "get-stream",
+    "chat",
+    "notification",
+    "feed",
+    "stream.io",
+    "getstream"
+  ],
```
Comment (ferhatelmas) [inline package.json:142]: could you remove comma ?

**Review by ferhatelmas — CHANGES_REQUESTED** [review]
it requires json, please remove last comma

File: package.json (hunk 1 above)
Comment (TylerAHolden) [inline package.json:142]:
```suggestion
  ]
```

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**